darts/models/forecasting/transformer_model.py

```diff
@@ -142,10 +156,43 @@ def __init__(
 
         raise_if_not(activation in FFN, f"'{activation}' is not in {FFN}")
         if activation in GLU_FFN:
+            raise_if(
+                custom_encoder is not None or custom_decoder is not None,
+                "Cannot use `custom_encoder` or `custom_decoder` along with an `activation` from "
+                f"{GLU_FFN}",
+                logger=logger,
+            )
             # use glu variant feedforward layers
             self.activation = getattr(glu_variants, activation)(
                 d_model=d_model, d_ff=dim_feedforward, dropout=dropout
             )
+            encoder_layer = _CustomFeedForwardEncoderLayer(
```

Comment:
the glu variants are full feedforward layers. It wasn't fully correct before to set the activation = glu_variants. 
For this to be fully correct, I think there should be a separate arg for ff_bock and leave activation alone. 

This customFeedForwardEncoderLayer makes more sense.